.eslintrc.js

```diff
@@ -25,6 +25,10 @@ module.exports = {
     es6: true,
   },
   rules: {
+    '@typescript-eslint/array-type': ['error', 'array-simple'],
```

Comment:
In Jest we have this set to `generic` for consistency

Reply:
consistency with what? (I don't care either way, and since this is fixable it really doesn't matter what we go with, as long as we lint for one or the other 🙂)

Reply:
My two-cents: I've found `array-simple` to look rather nice, and gives you the best of both worlds in terms of readability vs extra typing/characters/line length.

Just for the record, here's the difference:

```
// 'generic' - always use "Array", never "[]"
declare const arr: Array<string | number>;
declare const arr: Array<MyObj<string>>;
declare const arr: Array<string>;

// 'array-simple' - always use '[]', unless the generic type is "complex" (uses generics, union, etc)
declare const arr: Array<string | number>;
declare const arr: Array<MyObj<string>>;
declare const arr: string[];

// 'simple' - always use `[]`, never `Array`
declare const arr: (string | number)[];
declare const arr: MyObj<string>[];
declare const arr: string[];
```

I use to use `generic`, but got sick of having to go to the start of the type everytime I wanted to make thing like a `string` an array - w/ `array-simple` you can just keep typing when doing quick speedy type definitions.

Reply:
Just thought it'd be nice to have a code base that is easy to get into if you've already worked on Jest itself. Don't care much either, I always write it in `array` and let it autofix anyway 😄